### PR TITLE
Add listing of organization members

### LINKED ISSUE #377: Add listing of organization members
**Please do not report security vulnerabilities here**. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.

**Thank you in advance for helping us to improve this library!** Your attention to detail here is greatly appreciated and will help us respond as quickly as possible. For general support or usage questions, use the [Auth0 Community](https://community.auth0.com/) or [Auth0 Support](https://support.auth0.com/). Finally, to avoid duplicates, please search existing Issues before submitting one here.

By submitting an Issue to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

Currently, there's no way to list members of an organization.

### Describe the ideal solution

Add a `members` subcommand to the `orgs` command that would list members.

## Alternatives and current work-arounds

I'm not aware of any CLI alternatives. The only way to list members is through the API.

### Additional context

I'll submit a draft PR with the proposed solution for your consideration.

### PR DESCRIPTION
**Pull request by sergeybykov**
By submitting a PR to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/auth0/.github/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

Added `members` subcommand (`membersOrganizationCmd` func) to the `orgs` command to list members of an organiozation.
Added a `Members` method to `OrganizationAPI` interface to expose the underlying API.
Added `membersView` for rendering a list of members.
Refactored the pagination logic into a `getWithPagination` helper func to reuse the logic between `membersOrganizationCmd` and `listAppsCmd`. This required some ugly manipulation of slices due to the lack of generics. Maybe there's a better way to do this?

Fixes #377 

### References

Fixes #377.

### Testing

> Describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. If this library has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors.
>
> Please include any manual steps for testing end-to-end or functionality not covered by unit/integration tests.
>
> Also include details of the environment this PR was developed in (language/platform/browser version).

I tested this by executing the new cli subcommand against existing orgs.
I'm happy to add unit/integrations tests if somebody could guide me how this is done here.

- [ ] This change adds test coverage for new/changed/fixed functionality

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used, if not `master`

### CODE REVIEW

**Comment by Widcket** [comment]
Hi @sergeybykov, thanks for raising this. I haven't been able to take a look at the PR yet. When I have some bandwidth I'll circle back and review.

**Comment by Widcket** [comment]
Hi @sergeybykov, thanks for raising this. Is this PR ready for review?

**Comment by sergeybykov** [comment]
Hi @Widcket, yes it is ready. I only made it a draft because wanted to hear on the issue first.

**Comment by Widcket** [comment]
I'm busy right now, but I'll circle back when I have some bandwidth.

**Comment by sergeybykov** [comment]
Hi @Widcket, did you get a chance to take a look?

File: internal/cli/organizations.go [hunk 1]
```diff
@@ -422,6 +424,59 @@ func openOrganizationCmd(cli *cli) *cobra.Command {
...
+		Number int
+	}
+
+	cmd := &cobra.Command{
+		Use:   "members",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "List members of an organization",
+		Long:  "List members of an organization.",
+		Example: `auth0 orgs members 
+auth0 orgs members <id>`,
```
Comment (Widcket) [inline internal/cli/organizations.go:439]: This command should be `auth0 orgs member list <id>`. See https://auth0.github.io/auth0-cli/auth0_apis_scopes_list.html for a similar example (`auth0 apis scopes list <id>`).

File: internal/display/members.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+func (v *membersView) AsTableRow() []string {
+	return []string{ansi.Faint(v.ID), v.Name, v.Email, v.PictureURL}
+}
+
+func (v *membersView) KeyValues() [][]string {
+	return [][]string{
+		{"ID", ansi.Faint(v.ID)},
+		{"NAME", v.Name},
```
Comment (Widcket) [inline internal/display/members.go:29]:
```suggestion
		{"Name", v.Name},
```

File: internal/display/members.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+		{"Email", v.Email},
```
Comment (Widcket) [inline internal/display/members.go:30]: The keys for the key/value view should be in all caps. See https://github.com/auth0/auth0-cli/blob/main/internal/display/organizations.go#L33 as an example.

**Review by Widcket — CHANGES_REQUESTED** [review]
Thanks for this PR! Please have the command be `auth0 orgs members list <id>` and address the formatting issue in the ke/value list.

**Comment by sergeybykov** [comment]
Thanks for reviewing, @Widcket.
I believe I addressed your feedback.

**Comment by Widcket** [comment]
This is now out in [v0.11.4](https://github.com/auth0/auth0-cli/releases/tag/v0.11.4).

**Pull request merged**